src/libs/actions/FormActions.ts

```diff
@@ -0,0 +1,28 @@
+import Onyx from 'react-native-onyx';
+import {KeyValueMapping} from 'react-native-onyx/lib/types';
+import {PartialDeep} from 'type-fest';
+import {OnyxFormKey} from '../../ONYXKEYS';
+import {Form} from '../../types/onyx';
+import * as OnyxCommon from '../../types/onyx/OnyxCommon';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+type ExcludeDraft<T> = T extends `${infer R}Draft` ? never : T;
```

Comment:
Does that work?

```suggestion
type ExcludeDraft<T> = T extends `${string}Draft` ? never : T;
```

Reply:
both works , but you solution doesn't need "eslint-disable" so I will stick to that, thanks !